be/src/exec/workgroup/scan_task_queue.h

```diff
@@ -76,6 +76,9 @@ class ScanTaskQueueWithWorkGroup final : public ScanTaskQueue {
     size_t size() const override { return _total_task_num.load(std::memory_order_acquire); }
 
 private:
+    // Calculate the actual cpu used by all wg
+    void cal_wg_cpu_real_use_ratio();
```

Comment:
In order to be consistent with the other private function names, maybe it should be named to `_cal_wg_cpu_real_use_ratio`.